app/models/account.rb

```diff
@@ -1,3 +1,9 @@
 class Account < ApplicationRecord
   belongs_to :project
+  belongs_to :account_manager, class_name: 'User'
+
+  validates :name, presence: true
+  validates :services, presence: true, allow_nil: false
```

Comment:
Szerintem ha szerepel a `presence: true`, akkor ebben az esetben nincs szükségünk az `allow_nil: false`-ra.

Kérlek ellenőrizd le, hogy egy feltöltött jsonb tömb átmegy-e a validáción, de egy nem feltöltött, nem megy át. pl ezeknek nem szabadna átmennie.:
```ruby
''
'[]'
```

Valószínűleg ide vagy egy custom validációra lesz szükségünk, ami a string-et tömbbé alakítja, és megvizsgálja a hosszát. Vagy rávizsgáhatunk csak simán a string hosszára is, de az elsőre kicsit hacky-nek hangzik.